plugins/org.jboss.tools.locus.jsonschema2pojo/0.4.13/META-INF/MANIFEST.MF

```diff
@@ -1,66 +1,22 @@
-Bundle-Name: %pluginName
-Bundle-SymbolicName: org.jboss.tools.locus.jsonschema2pojo.jsonschema2pojo-core;singleton:=true
-Bundle-Version: 0.4.13.qualifier
-Bundle-ClassPath: .
-Bundle-Vendor: %providerName
-Bundle-Localization: plugin
-Require-Bundle: com.fasterxml.jackson.core.jackson-databind, 
- javax.validation
-Export-Package: org.jsonschema2pojo.DefaultGenerationConfig,
- org.jsonschema2pojo.CompositeAnnotator,
- org.jsonschema2pojo.exception.GenerationException,
- org.jsonschema2pojo.exception.ClassAlreadyExistsException,
- org.jsonschema2pojo.GsonAnnotator,
- org.jsonschema2pojo.FragmentResolver,
- org.jsonschema2pojo.SchemaGenerator,
- org.jsonschema2pojo.SchemaMapper,
- org.jsonschema2pojo.SourceType,
- org.jsonschema2pojo.AbstractAnnotator,
- org.jsonschema2pojo.Annotator,
- org.jsonschema2pojo.AnnotatorFactory,
- org.jsonschema2pojo.AllFileFilter,
- org.jsonschema2pojo.rules.TitleRule,
- org.jsonschema2pojo.rules.PrimitiveTypes,
- org.jsonschema2pojo.rules.PropertyRule,
- org.jsonschema2pojo.rules.DescriptionRule,
- org.jsonschema2pojo.rules.MediaRule,
- org.jsonschema2pojo.rules.RuleFactory,
- org.jsonschema2pojo.rules.ValidRule,
- org.jsonschema2pojo.rules.PropertiesRule,
- org.jsonschema2pojo.rules.AdditionalPropertiesRule,
- org.jsonschema2pojo.rules.PatternRule,
- org.jsonschema2pojo.rules.Rule,
- org.jsonschema2pojo.rules.SchemaRule,
- org.jsonschema2pojo.rules.RequiredRule,
- org.jsonschema2pojo.rules.MinItemsMaxItemsRule,
- org.jsonschema2pojo.rules.ObjectRule,
- org.jsonschema2pojo.rules.FormatRule,
- org.jsonschema2pojo.rules.MinLengthMaxLengthRule,
- org.jsonschema2pojo.rules.TypeRule,
- org.jsonschema2pojo.rules.RequiredArrayRule,
- org.jsonschema2pojo.rules.MinimumMaximumRule,
- org.jsonschema2pojo.rules.DefaultRule,
- org.jsonschema2pojo.rules.ArrayRule,
- org.jsonschema2pojo.rules.EnumRule,
- org.jsonschema2pojo.SchemaStore,
- org.jsonschema2pojo.DefaultGenerationConfig,
- org.jsonschema2pojo.URLProtocol,
- org.jsonschema2pojo.NoopAnnotator,
- org.jsonschema2pojo.Jsonschema2Pojo,
- org.jsonschema2pojo.Jackson1Annotator,
- org.jsonschema2pojo.FileCodeWriterWithEncoding,
- org.jsonschema2pojo.AnnotationStyle,
- org.jsonschema2pojo.util.NameHelper,
- org.jsonschema2pojo.util.Inflector,
- org.jsonschema2pojo.util.TypeUtil,
- org.jsonschema2pojo.util.RuleAndReplacement,
- org.jsonschema2pojo.util.URLUtil,
- org.jsonschema2pojo.util.ParcelableHelper,
- org.jsonschema2pojo.Schema,
- org.jsonschema2pojo.ContentResolver,
- org.jsonschema2pojo.GenerationConfig,
- org.jsonschema2pojo.Jackson2Annotator
-Bundle-RequiredExecutionEnvironment: JavaSE-1.6
-Eclipse-BuddyPolicy: registered
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: %pluginName
+Bundle-SymbolicName: org.jboss.tools.locus.jsonschema2pojo.jsonschema2pojo-core;singleton:=true
+Bundle-Version: 0.4.13.qualifier
+Bundle-ClassPath: .
+Bundle-Vendor: %providerName
+Bundle-Localization: plugin
+Require-Bundle: com.fasterxml.jackson.core.jackson-annotations,
```

Comment:
Hey @bfitzpat @lhein @maxandersen - I'm back - thanks for picking this issue up.
We're under some time pressure for getting locus unblocked (we have a code freeze this week).
Given that some exploration is needed can I suggest that we push these changes so I can get a SNAPSHOT TP built?  We can Jira a new issue for changing the Require-Bundle to Import-Package.